core/trino-main/src/main/java/io/trino/execution/scheduler/faulttolerant/EventDrivenFaultTolerantQueryScheduler.java

```diff
@@ -3071,6 +3071,7 @@ default T onSinkInstanceHandleAcquired(SinkInstanceHandleAcquiredEvent event)
             return onEvent(event);
         }
 
+        @SuppressWarnings("UnusedVariable")
```

Comment:
Don't suppress. I think you can just name the variable "ignored" to please the checker.